### ISSUE TITLE
PyPI page info needs updating

### ISSUE THREAD

**Issue by pllim** [iraf-community/pyraf#88]
https://pypi.org/project/pyraf/ still has old info from STScI. Info and metadata there needs to be updated to clarify that releases now come from `iraf-community`, not STScI.

**Comment by olebole** [comment]
I think that the information in PyPI come from the metadata and the README in the package itself, and this will be updated with a new package upload. I will check this before doing a new release, however feel free to review the current metadata & README.

**Comment by olebole** [comment]
One question here: There is a [Code of Conduct](https://github.com/iraf-community/pyraf/blob/9c4a7b829dc5cbecbcf32b1f318ebf12a94217fc/CODE_OF_CONDUCT.md) in the package. While I would like to keep it in principle, it refers to "Spacetelescope" projects. If you don't mind, I would remove that phrasing.

**Comment by pllim** [comment]
Yes, feel free to change the names.

**Comment by pllim** [comment]
@olebole , do you have a backup release manager for pyraf? STScI does not mind giving you control of the PyPI account but we are also worried about single point of failure.

**Comment by olebole** [comment]
No; I actually ***am*** a single point of failure (for the whole iraf-community ecosystem). I am a volunteer, and IRAF work is unpaid (to make that clear). Someone as a backup would be great.

**Comment by pllim** [comment]
@olebole , in that case @cdsontag is happy to stay on as PyPI maintainer, as discussed offline. FYI.
